test/functionalTest/cases/0000_ngsi10_convenience/subscriptions_onchange.test

```diff
@@ -38,8 +38,8 @@ payload='{
         }
     ],
     "attributes": [
-        "temperature",
-        "ligthstatus"
+        "ligthstatus",
```

Comment:
With the changes in https://github.com/telefonicaid/fiware-orion/compare/6eafb3f...7a31172 the "attributes" vector now defines the order of the attributes as they have to appear in the response. I see this as a little improvement, but the .test needs this slight adjust.